io.github.NickKarpowicz.LightwaveExplorer.json

```diff
@@ -0,0 +1,165 @@
+{
+    "id": "io.github.NickKarpowicz.LightwaveExplorer",
+    "branch": "master",
+    "runtime": "org.gnome.Platform",
+    
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions":["org.freedesktop.Sdk.Extension.llvm15"],
+    "add-extensions":{
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "directory": "lib/ffmpeg",
+            "version": "22.08",
+            "add-ld-path": "."
+        }
+    },
+    "command": "flatpakLauncher.sh",
+    "finish-args": [
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
if it supports waylandm then line 20 should have `--fallback-x11`

Reply:
good to know, I'll add it.